tools/git/test-affected-targets.sh

```diff
@@ -42,7 +42,7 @@ fi
 readarray -t TESTS < <(bazel query \
   --keep_going \
   --noshow_progress \
-  "tests(rdeps(//..., set($*))) except attr('tags', '$TAG_RE', //...)")
+  "tests(set(${TARGETS[*]}))")
```

Comment:
This removes the filtering of OMIT_TAGS

Reply:
Oh, I see.

Reply:
That filtering is done on `TARGETS` already. But fair that it wouldn't filter expanded `test_suite` rules.